README.md

```diff
@@ -1 +1,523 @@
+
+* [DAG-CBOR](https://ipld.io/specs/codecs/dag-cbor/spec/)
+* [UCAN Invocation](https://github.com/ucan-wg/invocation/)
+* [VarSig](https://github.com/ChainAgnostic/varsig/)
+
+# 0 Abstract
+
+An IPVM Workflow is a declarative configuration that extends a [UCAN Invocation](https://github.com/ucan-wg/invocation). A Workflow provides everything required to execute one or more tasks: defaults, tasks and their dependencies, authorization, metadata, signatures, and so on.
+
+# 1 Introduction
+
+> In late 1970 or early ’71 I approached IBM Canada’s Intellectual Property department to see if we could take out a patent on the basic idea [of dataflow]. Their recommendation, which I feel was prescient, was that this concept seemed to them more like a law of nature, which is not patentable. 
+>
+> J. Paul Morrison, [Flow-Based Programming](https://jpaulm.github.io/fbp/book.html)
+
+The potential complexity of a fully distributed execution by untrusted peers is very high. IPVM Workflows reduce the number of possible states by forcing explicit handling of any dangerous effects. The IPVM Workflow spec is a declarative document that MAY be inspected, transmitted, logged, and negotiated. Unlike s systems like WASI, there is a strict separation of effects from pure data, an emphasis on verifiability, and [promise pipelining](http://erights.org/elib/distrib/pipeline.html).
+
+IPVM Workflows MUST be suitable for the proposal of workflows and negotiation with providers on a discovery layer (ahead of credential delegation), execution on untrusted peer machines, and verification. Workflows SHOULD provide a sufficiently expressive base to build more complex models such as actors, event-driven systems, map-reduce, and so on.
+
+## 1.1 Design Philosophy
+
+While IPVM in aggregate is capable of executing arbitrary programs, individual IPVM Workflows are specified declaratively, and tasks workflows MUST be acyclic. Invocation in the declarative style liberates the programmer from worrying about explicit sequencing, parallelism, memoization, distribution, and nontermination in a trustless settings. Such constraints also grants the runtime control and flexibility to schedule tasks in an efficient and safe manner.
+
+These constraints impose specific practices. There is no first-class concept of persistent objects or loops. Loops, actors, vats, concurrent objects, and so on MAY be implemented on top of IPVM Workflows by enqueuing new workflows with the effect system (much like a [mailbox receive loop](https://www.erlang.org/doc/efficiency_guide/processes.html)).
+
+## 1.2 Foundational Authority
+
+IPVM workflows are built on top of cryptographic capabilities, providing a strong basis for distributed computation in trustless networks. This even provides a clear basis for crossing Web 2.0 and Web3 systems, other computation networks, local operation (in full or part).
+
+The IPVM Workflow spec extends on several other specs that have been developed to provide this basis:
```

Comment:
I know the frontmatter still uses `job` (i.e. https://github.com/ipvm-wg/spec), but it may be good to highlight why we're using `workflow` > `job`, especially when other systems like Bacalhau use `job` as well.

Reply:
> I know the frontmatter still uses job (i.e. https://github.com/ipvm-wg/spec)

Great point! It will not use the term "job" in the long-term. Consider that old text, which will get updated independently.